### PR TITLE
OSK update [semver:major]

### PR DESCRIPTION
**Pull request by KyleTryon**
## **New**
  - Utilizes the new OSK development pipeline.
  - New/Replaced integration tests for OSK
  - No longer checks for versions via the GitHub API which sometimes returns 403. 
    - Addresses #4
  - Examples added
    - 2x Examples, one for Chrome/ChromeDriver and one for Install-Browser-Tools
    - Addresses #5 

## **Removed**
  - **Debug parameters**.
    - Debug parameters were written for the convenience of the orb author but serve no value to the customer directly. When bugs are encountered we should create a branch for testing out and replicating that bug, then a new branch again for the fix.
  - **Removed Alpine Support**.
    - The Firefox installation for Alpine proved to be broken when testing this orb but was not reported by users. 
    - It was reported by us in 2019 there was an issue with GeckoDriver on Alpine, and no community reaction was seen in this issue.
    - Alpine appears to be unpopular in this orb and requires. Significant refactoring needed to support Alpine. If requested, we can add Alpine support back as a minor update in the future.

## Release Type: Major
**Why?**
  - Alpine support removed.
  - Will get us to a 1.0.0 release

### CODE REVIEW

File: src/commands/install-geckodriver.yml [hunk 1]
```diff
@@ -98,7 +88,7 @@ steps:
 
         GECKODRIVER_VERSION_NUM="$(echo $GECKODRIVER_VERSION | sed -E 's/v//')"
 
-        if geckodriver --version | grep "$GECKODRIVER_VERSION_NUM"<<^parameters.debug>> > /dev/null 2>&1<</parameters.debug>>; then
+        if geckodriver --version | grep "$GECKODRIVER_VERSION_NU > /dev/null 2>&1; then
```
Comment (felicianotech) [inline src/commands/install-geckodriver.yml:91]: > "$GECKODRIVER_VERSION_NU

Looks like this was cut too short. I think it's suppose to be:

`"$GECKODRIVER_VERSION_NUM"`

Comment (KyleTryon) [reply src/commands/install-geckodriver.yml:91]: Thank you, I'll have to dig in deeper as well to find out why that didn't seemingly break anything.

**Comment by KyleTryon** [comment]
> Made some comments.
> 
> I saw that the debug parameter was removed. Sounds good to me, but I'm confused as to what the final solution is to be? To way this PR is now, the parameter was removed but the code just assumes as if it was on. All the debug commands remain. Is that the intention?

The debug commands previously echoed to the terminal if enabled. Now output is always going to /dev/null. I have disabled the parameter and removed it effects. Users who did not enable this will see no change.

My reason for removing it is because this is not functional to the use of the orb, though maybe useful for debugging (duh), it serves mostly for the orb authors. I think we can debug this on our own without imposing the extra parameter on the user.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-_For full usage guidelines, see the [orb registry listing](http://circleci.com/orbs/registry/orb/circleci/browser-tools)._
+For full usage guidelines, see the [orb registry listing](http://circleci.com/orbs/registry/orb/circleci/browser-tools).
 
 ## Contributing
 
-We welcome [issues](https://github.com/CircleCI-Public/browser-tools-orb/issues) to and [pull requests](https://github.com/CircleCI-Public/browser-tools-orb/pulls) against this repository!
+We welcome [issues](https://github.com/CircleCI-Public/browser-tools/issues) to and [pull requests](https://github.com/CircleCI-Public/browser-tools/pulls) against this repository!
 
-For further questions/comments about this or other orbs, visit [CircleCI's orbs discussion forum](https://discuss.circleci.com/c/ecosystem/orbs).
```
[the remark this answers is not in the document]
Comment (KyleTryon) [reply README.md:13]: This was copied from another orb. It must also have the wrong URL as well. I'll try to search for those.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+# Browser-Tools Orb  [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/browser-tools-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/browser-tools-orb) [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/circleci/browser-tools)](https://circleci.com/orbs/registry/orb/circleci/browser-tools) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/browser-tools-orb/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
 
-Install common browsers and browser tooling on CircleCI for use in browser testing.
+Install a variety of browsers and tools for browser testing. Includes Chrome, FireFox, ChromeDriver and GeckoDriver.
```
[the remark this answers is not in the document]
Comment (KyleTryon) [reply README.md:3]: fair

File: src/commands/install-geckodriver.yml (hunk 1 above)
Comment (KyleTryon) [reply src/commands/install-geckodriver.yml]: Geckdriver testing was disabled previously because it was broken on alpine.

**Comment by KyleTryon** [comment]
gecko test re-enabled

**Pull request merged**